renderers/browser-render.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
```

Comment:
Could you remind me where this is used? I'm guessing in `mup-web` app code somewhere?